docs/scos/dev/glue-api-guides/202304.0/dynamic-data-api/how-to-guides/how-to-configure-dynamic-data-api.md

```diff
@@ -0,0 +1,110 @@
+---
+title: How to configure Dynamic Data API endpoints.
+description: This guide shows how to configure the Dynamic Data API endpoints.
+last_updated: June 23, 2023
+template: howto-guide-template
+---
+
+This guide shows how to configure the Dynamic Data API endpoints.
+
+In order to incorporate a new endpoint for interacting with entities in the database, 
+it is necessary to add a corresponding row to the `spy_dynamic_entity_configuration` table. 
+
+The `spy_dynamic_entity_configuration` table represents the configuration for dynamic entity endpoints in the system. It has the following columns:
+
+| COLUMN | SPECIFICATION |
+| --- | --- |
+| id_dynamic_entity_configuration | The unique identifier for the configuration. |
+| table_alias | An alias used in the request URL to refer to the endpoint. |
+| table_name | The name of the corresponding table in the database to operate on. |
+| is_active | Indicates whether the endpoint is enabled or disabled. |
+| definition | A JSON-formatted string containing the configuration details for each field in the table. |
+| created_at | Represents the date and time when the configuration was created. |
+| updated_at | Represents the date and time when the configuration was last updated. |
+
+{% info_block infoBox %}
+
+Currently, the process entails manually executing SQL queries as there is no existing user interface (UI) feature in Spryker for it. 
+However, future releases are expected to introduce UI solution for adding new rows to the `spy_dynamic_entity_configuration` table.
+
+{% endinfo_block %}
+
+Let's dive deeper into the configuration of the `spy_dynamic_entity_definition.definition` field.
+
+Below is an example snippet illustrating the structure of a possible definition value based on `spy_country` table:
+
+```json
+{
+  "identifier": "id_country",
+  "fields": [
+    { 
+      "fieldName": "iso2_code",
+      "fieldVisibleName": "iso2_code",
+      "type": "string", 
+      "isEditable": true,
+      "isCreatable": true,
+      "validation": {
+        "isRequired": true,
+        "maxLength": 2,
+        "minLength": 2
+      }
+    }
+  ]
+}
```

Comment:
https://github.com/spryker/spryker-docs/pull/1944#discussion_r1247915549 is not fixed.
The identifier field should also be defined in the `fields` array.